client/src/main.jsx

```diff
@@ -1,10 +1,18 @@
+"use client"
+
+import { ErrorBoundary } from "react-error-boundary";
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.jsx";
-import "./index.css";
+import {store} from './app/store.jsx'
+import { Provider } from "react-redux";
 
 ReactDOM.createRoot(document.getElementById("root")).render(
   <React.StrictMode>
+    <ErrorBoundary fallback={<div>Something went wrong</div>}>
```

Comment:
instead of the div, create external error component and import it. this will make it easier to edit it in the future

Reply:
done